### ISSUE TITLE
Why is it required for "everyone" to be on Targeted Release for this to work?

### ISSUE THREAD

**Issue by jmuehlenbein** [pnp/sp-starter-kit#431]
- [x] Question

Hello, I'm trying to use the stock web part but the requirement to have everyone on Targeted Release is a deal breaker.  Is there any way around this requirement? Can you help me connect the dots to understand why this is required?

**Comment by bcameron1231** [comment]
This isn't actually a requirement anymore. It was a requirement of v1 of the starter kit when it was initially released to use the Graph Web Parts.. We'll update the documentation to reflect this.

You are good to go to use this! I'm going to close this out :)

**Comment by jmuehlenbein** [comment]
Are you sure because I tried to use the stock info webpart with tenant set to Enterprise release and it didn’t work until I switched it Targeted release for Everyone. I will test again….

**Comment by jmuehlenbein** [comment]
I guess it must be something other than the release cycle causing the issue. It’s not working in either case now. The time updates but the stock price doesn’t.  =(  Maybe a network thing.  This SPFx is so overwhelming!

Joy Muehlenbein | Digital Workspace Architect | Enterprise Technology - Engineering
Edwards Lifesciences | www.edwards.com<http://www.edwards.com/> | One Edwards Way | Irvine, CA 92614
mobile: +1.858.405.7185
Joy_Muehlenbein@edwards.com

From: Joy Muehlenbein
Sent: Thursday, August 27, 2020 8:38 AM
To: pnp/sp-starter-kit <reply@reply.github.com>
Subject: RE: [EXTERNAL] Re: [pnp/sp-starter-kit] Why is it required for "everyone" to be on Targeted Release for this to work? (#431)

Are you sure because I tried to use the stock info webpart with tenant set to Enterprise release and it didn’t work until I switched it Targeted release for Everyone. I will test again….
